CHANGELOG.next.asciidoc

```diff
@@ -44,6 +44,10 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Update Salesforce module to use new Salesforce input. {pull}37509[37509]
 - Tag events that come from a filestream in "take over" mode. {pull}39828[39828]
 - Fix high IO and handling of a corrupted registry log file. {pull}35893[35893]
+- Journald: removed configuration options `include_matches.or`, `include_matches.and`, `backoff`, `max_backoff`, `cursor_seek_fallback`. {pull}40061[40061]
+- Journald: `include_matches.match` now behaves in the same way as matchers in `journalctl`. {pull}40061[40061]
+- Journald: `seek` and `since` behaviour have been simplified, if there is a cursor (state) `seek` and `since` are ignored and the cursor is used. {pull}40061[40061]
+- Filebeat now needs to use `exec` and `fork` syscalls to run the journald input. If this input is not being used, the syscalls are not needed. {pull}40061[40061]
```

Comment:
I wasn't sure the best way to add so much information to the breaking changes bit, so I split them into multiple entries.

Reply:
Multiple entries is fine, it's a list of changes not a list of PRs.